src/JsConverter.php

```diff
@@ -280,4 +280,15 @@ protected static function fixQuoteEscapingForSingleQuoteDelimitedStrings(array &
             $string = preg_replace("/\\\\'/", "'", $string);
         }
     }
+
+    /**
+     * Removes oneline and multiline comments from JSON string
+     *
+     * @param string $jsonString
+     * @return string
+     */
+    protected static function testRemoveComments(string $jsonString): string
```

Comment:
Is there a reason to be named `testRemoveComments` instead of simply `removeComments`?

Reply:
my bad.